src/PlayerSkeleton.java

```diff
@@ -25,3 +26,204 @@ public static void main(String[] args) {
 	}
 	
 }
+
+class Heuristics{

```

Comment:
maybe can add a space before the bracket?

Reply:
done